<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeUI</class>
 <widget class="QMainWindow" name="EmployeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>395</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonSubmit">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>281</width>
      <height>222</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDateDebut">
       <property name="text">
        <string>Date de Debut</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEditDebut"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDateFin">
       <property name="text">
        <string>Date de fin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEditFin"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditStatus">
       <property name="font">
        <font>
         <family>Myanmar Text</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Motif de conge</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableViewUsers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::CustomDashLine</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>82</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButtonProfil">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Page Demande de Conge </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>70</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Formulaire de demande de conge</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
